### DISCUSSION TITLE
How do you remove sidebar on homepage?

### DISCUSSION THREAD

**Discussion by erondpowell** [vuejs/vitepress#832] [Q&A] [answered]
**I noticed that on the VitePress website there is no sidebar visible on the homepage:**
\
<img width="1719" alt="Screen Shot 2022-06-21 at 10 42 52 AM" src="https://user-images.githubusercontent.com/62306630/174841552-805feefb-7c00-40f7-a3dc-6f95cd4d3976.png">
\
\
**However, if I copy your site's Homepage frontmatter from Github, I still get a sidebar:**
\
<img width="1728" alt="Screen Shot 2022-06-21 at 10 47 15 AM" src="https://user-images.githubusercontent.com/62306630/174842479-64556d52-c1b4-476e-a154-523fde697b5c.png">
\
**I looked through the the documentation as well as the VitePress website Github repo's `index.md` and `config.js` files and I'm unable to find any info on how VitePress toggled off the sidebar for their Homepage. How can I get this result?**
\
\
**I am happy to make a PR to add documentation, if relevant.**

**Comment by kiaking** [comment, marked as answer]
This is handled. https://github.com/vuejs/vitepress/pull/829

From next release, Home page will not have sidebar 👍
